meetings/2022-04-07.md

```diff
@@ -0,0 +1,174 @@
+# W3C Solid Community Group: Notifications Panel
+
+* Date: 2022-04-07T14:00:00Z
+* Call: https://meet.jit.si/solid-notifications
+* Chat: https://gitter.im/solid/notifications-panel
+* Repository: https://github.com/solid/notifications-panel
+* Status: Published
+
+
+## Present
+* [Sarven Capadisli](https://csarven.ca/#i)
+* Aaron Coburn
+* Jackson Morgan
+* elf Pavlik
+
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Aaron Coburn
+
+### Introductions
+* name: text
+
+
+---
+
+## Topics
+
+
+### WIP Implementation Feedback
+* SC: We'll allocate some time for implementation feedback going forward. Links to servers/applications welcome.
+* (none provided)
+
+### Notifications Protocol and WebSocket Subscription Type PRs
+* SC: Proposals to add technical reports TR/2022/notifications-protocol-20220401 and TR/2022/websocket-subscription-2021-20220401:
+ * https://github.com/solid/specification#391
+ * https://github.com/solid/specification#392
+* SC: Updates can be done at source in the Editor's Drafts at https://github.com/solid/notifications . The snapshots that need to flow back to https://github.com/solid/specification can be done at:
+ * https://github.com/solid/notifications/tree/TR/2022/notifications-protocol-20220401
+ * https://github.com/solid/notifications/tree/TR/2022/websocket-subscription-2021-20220401
+* SC: Some members of the Editors Team will review next week and afterwards. (SC will abstain in this review as an editor)
+* SC: Questions or comments?
+* eP: re Snapshots -- the date is April 1, what is the meaning of that date? Should we reconsider the date in the snapshot
+* AC: Snapshot is from April 1st, if they are changes that are suggested, any of those changes should go back into underlying specification, they should be opened as issues and resolved via PR. Than they would be another snapshot let's say April 17th. If there are blocking reviews on April 1st it would be closed and not merged.
```

Comment:
```suggestion
* AC: Snapshot is from April 1st, if changes are suggested, those changes should go back into underlying specification; they should be opened as issues and resolved via PR. Than there would be another snapshot, let's say April 17th. If there are blocking reviews on April 1st, it would be closed and not merged.
```